app/pubspec.lock

```diff
@@ -1293,6 +1293,14 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "3.16.7"
+  multiavatar:
```

Comment:
Same here, this changes aren't related to actual PR.

Reply:
but the pubspec.lock needs to go into the repo or it won't properly resolve